### PR TITLE
added run_header schema

### PR DESCRIPTION
**Pull request by gwbischof**

### CODE REVIEW

File: event_model/schemas/run_header.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            "items": {
+                "$ref":"./event_descriptor.json"}},
+        "rescources": { 
+            "type":"array", 
+            "items": {
+                "$ref":"./resource.json"}},
+        "run_stop": {
+            "$ref": "./run_stop.json" }
+    },
+    "required": ["run_start"],
```
Comment (tacaswell) [inline event_model/schemas/run_header.json]: uid should be required too?

**Comment by tacaswell** [comment]
I am not sure we want to put this schema (which is an implementation detail of the mongo backing store) and this should live there.

On the other hand, I think that makes the references much harder to manage....

File: event_model/schemas/run_header.json (hunk 1 above)
Comment (gwbischof) [reply event_model/schemas/run_header.json]: I added it

**Comment by danielballan** [comment]
> On the other hand, I think that makes the references much harder to manage....

Looks like jsonschema supports HTTP references, so we can referenced the file in a tagged commit on GH.

@gwbischof I'm going to create a new repo suitcase-mongo for this.

**Comment by danielballan** [comment]
To be continued in http://github.com/NSLS-II/suitcase-mongo

**Pull request closed without merging**